sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/tables/InterfaceGeneration.kt

```diff
@@ -132,7 +148,20 @@ class InterfaceGeneration {
       |        override val floatValue: kotlin.Float,
       |        override val doubleValue: kotlin.Double,
       |        override val blobValue: kotlin.ByteArray
-      |    ) : com.example.Test
+      |    ) : com.example.Test {
+      |        override fun toString(): kotlin.String = ""${'"'}
+      |        |Test.Impl [
+      |        |  intValue: ${"$"}intValue
+      |        |  intValue2: ${"$"}intValue2
+      |        |  booleanValue: ${"$"}booleanValue
+      |        |  shortValue: ${"$"}shortValue
+      |        |  longValue: ${"$"}longValue
+      |        |  floatValue: ${"$"}floatValue
+      |        |  doubleValue: ${"$"}doubleValue
+      |        |  blobValue: ${"$"}blobValue
```

Comment:
Can you check what the normal toString does for arrays? We might need to call contentToString() on them.